tests/lib_files_test.py

```diff
@@ -111,11 +111,21 @@ def test_srw_delete(fc):
         "downloadFile",
```

Comment:
I don't think we need to test the deprecated API.

Reply:
[729f6ca](https://github.com/radiasoft/sirepo/pull/6348/commits/729f6ca092068ceff3cd01d549e71dd64024cbeb)